### ISSUE TITLE
📦Use webpack instead of rollup

### ISSUE THREAD

**Issue by jperl** [qawolf/qawolf#592]
It will require less dependencies

**Comment by Olyno** [comment]
You really needn't to change. Rollup is popular too, and its creator explain itself when Rollup should be used and when Webpack should.

> Use webpack for apps, and Rollup for libraries

**Source:** https://medium.com/webpack/webpack-and-rollup-the-same-but-different-a41ad427058c

If I quote qawolf's description

> QA Wolf is a Node.js library for creating browser tests.

**Comment by jperl** [comment]
Right now we use rollup for bundling the web parts of QA Wolf, not for the library. I ran into issues with rollup's [node polyfills](https://github.com/ionic-team/rollup-plugin-node-polyfills) not having promisify in another project. This plugin does not seem maintained, versus Webpack which has those built in since it is aimed for the web. Playwright also uses webpack for their web scripts so we know those will be bundlable with webpack. I just wanted to log this here in case we ran into issues. Right now we do not have any, so it is not a priority.
